explainaboard/info.py

```diff
@@ -191,13 +184,13 @@ def copy(self) -> "SysOutputInfo":
 
 @dataclass
 class OverallStatistics:
-    sys_info: SysOutputInfo = None
+    sys_info: Optional[SysOutputInfo] = None
     metric_stats: Any = None
-    active_features: list[str] = None
-    overall_results: dict[str, Performance] = None
+    active_features: Optional[list[str]] = None
+    overall_results: Optional[dict[str, Performance]] = None
 
 
 @dataclass
 class FineGrainedStatistics:
-    samples_over_bucket: dict = None
```

Comment:
I think these two are not actually optional.

Reply:
Fixed: 4c127d6